src/hooks/js/useAudio.js

```diff
@@ -0,0 +1,139 @@
+import { useEffect, useState } from "react";
+
+export const useAudio = (ref) => {
+  const audio = ref.current;
+
+  const [audioState, setAudioState] = useState({
+    isPaused: audio ? audio.paused : true,
+    isMuted: audio ? audio.muted : false,
```

Comment:
isMuted does not update its value when I mute the audio using the native controls